OneSignalExample/ProjectSettings/AndroidResolverDependencies.xml

```diff
@@ -1,10 +1,10 @@
 <dependencies>
   <packages>
-    <package>com.android.support:customtabs:[26.0.0, 27.2.0)</package>
-    <package>com.android.support:support-v4:[26.0.0, 27.2.0)</package>
-    <package>com.google.android.gms:play-services-base:[10.2.1, 12.1.0)</package>
-    <package>com.google.android.gms:play-services-location:[10.2.1, 12.1.0)</package>
-    <package>com.google.firebase:firebase-messaging:[10.2.1, 12.1.0)</package>
+    <package>com.android.support:customtabs:[26.0.0, 27.2.0[</package>
```

Comment:
shouldn't this be closing bracket?

Reply:
It should be open, it is to indicate exclusive instead of inclusive.
The change was done in here but the example project didn't have it's "lockfile" updated until this PR.
PR of the orignal change:
https://github.com/OneSignal/OneSignal-Unity-SDK/pull/152/files